### ISSUE TITLE
Run Terraform task: 'Error: "features": required field is not set'

### ISSUE THREAD

**Issue by KenSpur** [MicrosoftDocs/mslearn-tailspin-spacegame-web-automate#4]
Hey guys,

I am following the https://docs.microsoft.com/en-us/learn/modules/provision-infrastructure-azure-pipelines/8-run-terraform-pipeline module and bumped into an isseu running the Pipeline.

Currently as is the pipeline seems to fail at the Run Terraform task with the error: **"features": required field is not set.**

Adding 
```
provider "azurerm" {
  version = "=2.0.0"
  features {}
}
```
To the **main.tf** seems to fix it, which i found here : https://github.com/terraform-providers/terraform-provider-azurerm/issues/5867

Just wanted to point this out :), though can't say i know what i'm doing yet ^^

Btw I absolutely love what you guys are doing! **<3** Keep up the awesome work!

**Comment by woeterman94** [comment]
Thanks for reporting. Any other breaking changes?

**Comment by KenSpur** [comment]
Nope, that's it for https://docs.microsoft.com/en-us/learn/modules/provision-infrastructure-azure-pipelines/index
